sandy/core/endf6.py

```diff
@@ -1690,6 +1690,8 @@ def get_errorr(self,
             derived cross section energy bounds (default is `[1e-5, 2e7]`)
         iwt : `int`, optional
             weight function option (default is 2, constant)
+        mfcov : `int`
```

Comment:
Change mfcov as suggested below.
Keep consistency between groupr and errorr.
Don't let the user choose, activate mf31/34/35 if found in the endf6 file